WindowsServerDocs/manage/windows-admin-center/azure/azure-extended-network.md

```diff
@@ -0,0 +1,184 @@
+
+4. Azure Extended Network scans the network to discovered IP addresses and populates the Suggestion lists based on this scan.  To extend these addresses you must use the dropdown list and select the checkbox next to the discovered address.  Not all addresses will be discovered.  Optionally, use the **Add** button to manually add addresses that are not discovered automatically.
+![Image](../media/azure-extended-network/add-ipv4-addresses-panel-filled.png)
+
+5. Click **Submit** when complete.  You will see the status change to **Updating**, then **Progressing**, and finally back to **OK** when the configuration is complete.
+
+Your addresses are now extended.  Use the Add IPv4 Addresses button to add additional addresses at any time.  If an IP address is no longer in use at either end of the extended network, select the checkbox next to it and select Remove IPv4 Addresses.
+
+If you no longer want to use Azure Extended Network, click the **Remove Azure Extended-Network** button.  This will uninstall the agent from the two virtual appliances and remove the extended IP addresses.  The network will stop being extended.  You will have to re-run the setup after removing if you want to start using the extended network again.
+
+## Troubleshooting
+
+If you receive an error during the deployment of Azure Extended Network, follow the following steps:
+
+1. Verify that both virtual appliances are using Windows Server 2019.
+2. Verify that you are not running the Windows Admin Center gateway on one of the virtual appliances.  It is recommended that you run Windows Admin Center from an on-premises network.
+3. Make sure you can remote into the on-premises VM from the Windows Admin Center gateway using enter-pssession.
+4. If there is a firewall in between Azure and on-premises, confirm that it is configured to allow UDP traffic on the selected port (default 4789).   Use a tool such as ctsTraffic to configure a listener and a sender.  Verify traffic can be sent in both directions on the specified port.
+5. Use pktmon to verify that packets are being sent and received as expected.  Run pktmon on each virtual appliance:
+
+```powershell
+Pktmon start –etw
+```
+
+6. Run Azure Extended Network configuration
+
+```powershell
+Pktmon stop
+Netsh trace convert input=<path to pktmon etl file>
+```
+
+7. Open the text file that is produced from each virtual appliance and search for UDP traffic on the specified port (default 4789).   If you see traffic sent from the on-premises virtual appliance, but not received by the Azure virtual appliance then you need to verify the routing and firewall between the appliances.  If you see traffic sent from on-premises into Azure, you should then see the Azure virtual appliance send a packet in response.  If that packet is never received by the on-premises virtual appliance then you need to verify that the routing is correct and there is not a firewall blocking traffic in between.
```

Comment:
surplus space + readability comma instead of "then" (slightly overused)
```suggestion
7. Open the text file that is produced from each virtual appliance and search for UDP traffic on the specified port (default 4789).   If you see traffic sent from the on-premises virtual appliance, but not received by the Azure virtual appliance, you need to verify the routing and firewall between the appliances.  If you see traffic sent from on-premises into Azure, you should then see the Azure virtual appliance send a packet in response.  If that packet is never received by the on-premises virtual appliance, you need to verify that the routing is correct and there is not a firewall blocking traffic in between.
```